lxd/cluster/membership.go

```diff
@@ -457,15 +457,15 @@ func Join(state *state.State, gateway *Gateway, networkCert *shared.CertInfo, se
 	})
 
 	// If we are listed among the database nodes, join the raft cluster.
-	var info *db.RaftNode
+	var info db.RaftNode
 	for _, node := range raftNodes {
 		if node.Address == localClusterAddress {
-			info = &node
+			info = node
 		}
 	}
 
-	if info == nil {
-		panic("Joining member not found")
+	if (db.RaftNode{}) == info {
```

Comment:
what was the linter error here?

Reply:
I can't actually grab the rule for you at the moment since my LXD environment is busted but I recall the error was about assigning to a pointer of a loop var, which always has the same address.

Reply:
`range-val-address`